pkg/registry/registry.go

```diff
@@ -133,6 +135,13 @@ func (reg *Registry) RequestLogger() func(next http.Handler) http.Handler {
 				return
 			}
 
+			for _, path := range reg.AccessLogIgnoredPaths {
+				if r.URL.Path == path {
+					next.ServeHTTP(w, r)
+					return
+				}
```

Comment:
```go
if slices.Contains(reg.AccessLogIgnoredPaths, r.URL.Path) {
    next.ServeHTTP(w, r)
    return
}
```

Requires `golang.org/x/exp/slices` and currently not a stable API.

Reply:
This was adressed and fixed in #149